rfcs/0014-task-workers.md

```diff
@@ -177,20 +192,23 @@ The `processors.execute` function shall accept the following parameters:
 
 The `execute` function shall validate that `resources` only contains `@ui5/fs/Resource` instances and that `options` adheres to the requirements stated in [Serializing Data](#serializing-data).
 
+#### Task Example
+
 ```js
 /**
  * Custom task example
  *
  * @param {Object} parameters Parameters
  * @param {DuplexCollection} parameters.workspace DuplexCollection to read and write files
  * @param {AbstractReader} parameters.dependencies Reader or Collection to read dependency files
+ * @param {@ui5/project/build/helpers/TaskUtil|object} [parameters.taskUtil] TaskUtil
+ * @param {object} processors
  * @param {Object} parameters.options Options
  * @param {string} parameters.options.projectName Project name
  * @param {string} [parameters.options.configuration] Task configuration if given in ui5.yaml
- * @param {object} processors
  * @returns {Promise<undefined>} Promise resolving with undefined once data has been written
  */
-module.exports = function({workspace, options, processors}) {
+module.exports = function({workspace, taskUtil, processors, options}) {
```

Comment:
This file can also be an ES module, right?

Reply:
Sure! Updated 👍